.changeset/early-feet-lay.md

```diff
@@ -0,0 +1,17 @@
+---
+'@backstage/plugin-auth-backend-module-oidc-provider': minor
```

Comment:
```suggestion
'@backstage/plugin-auth-backend-module-oidc-provider': patch
```

Reply:
Should be a patch as we're on 0.x of this package

Reply:
Hi @benjdlambert, thanks for your comment. Just fixed it.